ot/lowrank.py

```diff
@@ -0,0 +1,289 @@
+"""
+Low rank OT solvers
+"""
+
+# Author: Laurène David <laurene.david@ip-paris.fr>
+#
+# License: MIT License
+
+
+import warnings
+from .utils import unif, get_lowrank_lazytensor
+from .backend import get_backend
+
+
+def compute_lr_sqeuclidean_matrix(X_s, X_t, nx=None):
+    """
+    Compute low rank decomposition of a sqeuclidean cost matrix.
+    This function won't work for other metrics.
+
+    See "Section 3.5, proposition 1" of the paper
+
+    References
+    ----------
+    .. [63] Scetbon, M., Cuturi, M., & Peyré, G (2021).
```

Comment:
It would be better to put the reference of the published paper.

Scetbon, M., Cuturi, M., & Peyré, G. (2021). "Low-rank Sinkhorn factorization". In International Conference on Machine Learning.

Also the number [63] should be updated after the merge, as new references have been included and [63] relates to another paper.